labels-owners.yaml

```diff
@@ -35,6 +36,7 @@ labels:
     owners:
       - animeshsingh
       - cliveseldon
+      - cvenets
```

Comment:
@yanniszark what are you and @cvenets working on with respect to inference? I think inference is mostly kfserving.

Reply:
@cvenets has been working on PVC support for KFServing.
However, discussions have been made mainly in community meetings and not GitHub.
I'm removing him for now for this PR to go forward.